test/runnable/testpdb.d

```diff
@@ -42,7 +42,9 @@ void main(string[] args)
         S18984 s = test18984(session, globals);
 
         test19307(session, globals);
-        
+
+        test19318(session, globals);
```

Comment:
Also added in https://github.com/dlang/dmd/pull/9092 which didn't make the cut for stable. It would be good to cherry-pick it to stable before https://github.com/dlang/dmd/pull/8906 is released.

Reply:
Ah okay, closing then.